lib/Runtime/Language/TaggedInt.inl

```diff
@@ -206,7 +206,23 @@ namespace Js
     }
 #endif
 
+    // Returns true when the provided args only contains TaggedInts. Note that
+    // iteration starts from 0 (to account for 'this' at index 0).
+    // ** Is this too large of a function to inline?
```

Comment:
It shouldn't be too large, especially with PGO.

---
In reply to: [132061017](https://github.com/Microsoft/ChakraCore/pull/3495#discussion_r132061017) [](ancestors = 132061017)